meson.build

```diff
@@ -94,6 +94,15 @@ nasm_system_flags = [
 default_c_flags = []
 default_cpp_flags = []
 
+add_project_arguments([
+    '-Wl,-z,relro',
+    '-Wl,-z,noexecstack',
+    '-pie',
```

Comment:
pie can also be handled via meson's `pie: true` per-target kwarg override or the global `-D b_pie=true` setting.

Reply:
Interesting, I didn't know about `pie: true`. It might not have been supported with Meson 0.56, which we still needed due to a CentOS 7 dependency. We don't any more, so we can modernise.

Reply:
Docs say added in 0.49 :)